crates/cdk/src/publish.rs

```diff
@@ -52,6 +53,10 @@ pub struct PublishCmd {
 
     #[clap(long, hide_short_help = true)]
     remote: Option<String>,
+
+    /// Relative path to this connector package README
+    #[clap(long)]
+    readme: Option<PathBuf>,
```

Comment:
why this is optional? by default, we should add README

Reply:
if it is not specified, it is taking the README from the same directory where the Connector.toml file is

Reply:
`readme` is optional for the CLI, but by default CDK will find the README in the same directory as the `Connector.toml`.

If we happen to have a Readme in other directory, then we can overwrite the default behavior using the `readme` option.

If the readme is not present/found CDK will complain.

Reply:
if it is mandatory then we should make it so and provide default here instead in the GitHub Action

Reply:
For the CLI is not mandatory, what we do is:
1. If user provides a readme path then we use that
2. If the user doesn't provides a readme path, then we perform the current approach, which is checking for the README.md in the same path as the `Connector.toml`

Here is the code in question:

https://github.com/infinyon/fluvio/blob/0af53e3044be35acecd908e64c15bd63eed40203/crates/cdk/src/publish.rs#L157-L164